ndsi/network.py

```diff
@@ -185,6 +185,62 @@ def running(self):
         return bool(self.pyre_node)
 
 
+class Network:
+    def __init__(self, context=None, name=None, headers=(), callbacks=()):
+        self.context = context or zmq.Context()
+        self._callbacks = callbacks
+        self._nodes = [
+            NetworkNode(
+                format=format,
+                context=self.context,
+                name=name,
+                headers=headers,
+                callbacks=self._callbacks,
+            )
+            for format in DataFormat.supported_formats()
+        ]
```

Comment:
The user should be able to choose, which versions he want's the `Network` instance to support.